app/1.0/blog/2016-09-09-polymer-2.0.md

```diff
@@ -0,0 +1,95 @@
+---
+title:  "Polymer 2.0 Preview"
+---
+
+Today we pushed a [`2.0-preview`](https://github.com/polymer/polymer/tree/2.0-preview) branch to the Polymer repository, where we’re working on the next major version of the library.
+
+Polymer 2.0 is a minimally-breaking change from the current library, with a few very precise goals:
+
+### Take advantage of native [v1 Shadow DOM](https://developers.google.com/web/fundamentals/primers/shadowdom/) and [v1 Custom Elements](https://developers.google.com/web/fundamentals/primers/customelements/) API’s
+
+The primary goal of the Polymer 2.0 release is to take advantage of native, cross-browser support for Web Components.
+
+   Polymer 1.x is built on top of the so-called "v0" Web Components specs, which are supported natively only in Google Chrome; using Polymer in other browsers has always required the use of polyfills.
+
+   Beginning this fall, multiple browsers will be shipping native implementations of the new "v1" specs for Shadow DOM and Custom Elements, yielding better web components performance and reducing the need for polyfills.
```

Comment:
Hm what links to you think might be best to include? I'm thinking https://www.chromestatus.com/features/4667415417847808 and https://developer.apple.com/library/prerelease/content/releasenotes/General/WhatsNewInSafari/Articles/Safari_10_0.html#//apple_ref/doc/uid/TP40014305-CH11-DontLinkElementID_5 but are there good ones for Edge/Firefox?